prow/bugzilla/client.go

```diff
@@ -89,6 +91,59 @@ func NewClient(getAPIKey func() []byte, endpoint string) Client {
 	}
 }
 
+// NewBugDetailsCache is a constructor for bugDetailsCache
+func NewBugDetailsCache() *BugDetailsCache {
+	return &BugDetailsCache{cache: map[int]cacheEntry{}}
+}
+
+type cacheEntry struct {
+	bug       *Bug
+	isVisited bool
+}
+
+// BugDetailsCache holds the already retrieved bug details
+type BugDetailsCache struct {
```

Comment:
Nothing of the cache should be exported, this is a private implementation detail